test/CellDataTests/CellFieldsTests.jl

```diff
@@ -204,36 +204,76 @@ Random.seed!(0)
     domain = repeat([xmin, xmax], D)
     ncells = 3
     partition = repeat([ncells], D)
-    model = CartesianDiscreteModel(domain, partition)
-    # TODO: test both with and without this
-    model = simplexify(model)
-
+    base_model = CartesianDiscreteModel(domain, partition)
     order = 2
     reffe = ReferenceFE(lagrangian, Float64, order)
-    V = FESpace(model, reffe)
-
-    coeff0 = rand(Float64)
-    coeffs = rand(SVector{D,Float64})
-    f(x) = coeffs ⋅ SVector(Tuple(x)) + coeff0
-    # TODO: use this mechanism instead to project
-    # Francesc Verdugo @fverdugo 13:11
-    # a(u,v) = ∫( u*v )dΩ
-    # l(v) = a(f,v)
-    # Solve a fe problem with this weak form
-    # See also tutorial 10, "Isotropic damage model", section "L2
-    # projection", function "project"
-    fh = interpolate_everywhere(f, V)
-    fhcache = return_cache(fh, VectorValue(zeros(D)...))
-
-    xs = [VectorValue(rand(D)...) for i in 1:10]
-    for x in xs
-        x = VectorValue(rand(D)...)
-        fx = f(x)
-        fhx = evaluate!(fhcache, fh, x)
-        @test fhx ≈ fx
+
+    for M ∈ ["Hypercubes", "Simplices"]
```

Comment:
I would use symbols instead of strings:
```Julia
for M \in [:hypercubes, :simplices]
...
M === :hypercubes ? ...
```

Reply:
Fixed this. Waiting for @ericneiva to review the other changes.